packages/vite/src/utils.ts

```diff
@@ -29,3 +28,18 @@ function hash(keyParts: Array<string | NodeJS.ArrayBufferView>) {
   }
   return hash.digest('hex')
 }
+
+export const generateCacheID = (path: string) => createHash('sha1').update(path).digest('hex')
```

Comment:
This looks almost exactly the same as the hash function right above it. Could we figure out how to just have one function for this?

Reply:
I removed "generateCacheID" in favor of using your "hash" function.